dev/src/webpack/excludeNodeModulesAndDependencies.ts

```diff
@@ -43,7 +43,7 @@ export function excludeNodeModulesAndDependencies(
     //   so they can be found through normal require calls.
     // tslint:disable-next-line: strict-boolean-expressions
     webpackConfig.plugins = webpackConfig.plugins || [];
-    Object.assign(webpackConfig.plugins, new CopyWebpackPlugin(copyEntries));
+    webpackConfig.plugins.push(new CopyWebpackPlugin(copyEntries));
```

Comment:
Does it work if there are two CopyWebpackPlugins with different copyEntries? Since you've got this here and one recently added in this cosmos PR... https://github.com/Microsoft/vscode-cosmosdb/pull/1008/files

Reply:
Not a problem, thanks for asking.